gef-ui/api-test.jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  preset: 'ts-jest/presets/js-with-babel',
```

Comment:
`js-with-ts`?

Reply:
gef-ui uses Babel so I think it's needed here - can give it a try with `js-with-ts` if you  like?

Reply:
unit tests fail without it

Reply:
nope -- lets keep